models_config.truefoundry.yaml

```diff
@@ -9,6 +9,12 @@ model_providers:
       embedding_model_ids:
           - "openai-main/text-embedding-ada-002"
       reranking_model_ids: []
+      headers:
+          - "X-TFY-METADATA":
+                {
+                    "tfy_log_request": "true",
+                    "Custom-Metadata": "Cognita-LLM-Request",
+                }
```

Comment:
Why is this a list?
this should just be `Dict[str, str]`

Reply:
Like how should the yaml format be?